internal/applicationsnapshot/input.go

```diff
@@ -0,0 +1,61 @@
+package applicationsnapshot
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/hacbs-contract/ec-cli/internal/utils"
```

Comment:
Imports are not sorted. Ensure that imports from hacbs-contract are placed last. Run `make lint-fix` or manually correct.